.copr/Makefile

```diff
@@ -5,7 +5,7 @@ srpm:
 	dnf install -y git make curl gzip tar rpm-build golang fedpkg which
 
 	# Generate SRPM
-	make -C .. rpm/source RPM_BUILDER=fedpkg RELEASE=epel7
```

Comment:
A small heads up in case this gets merged too fast: The build target was removed along with the directory flag.
Right now it would default to running the "build" target. I don't know if this is expected behavior.
In case rpm/source is still the desired target you need to add it back in:
`make rpm/source RPM_BUILDER=fedpkg RELEASE=epel7`